test/classes/initializers/postInit/conditional.chpl

```diff
@@ -0,0 +1,32 @@
+
+config const flag = true;
+
+class A {
+  var a : int;
+  proc init() {
+    writeln("A.init");
+  }
+  proc postInit() {
+    writeln("A.postInit");
+  }
+}
+
+class B : A {
+  var b : bool;
+  proc init(b:bool) {
+    writeln("B.init");
+    this.b = b;
+  }
+  proc postInit() {
+    if b {
+      writeln("B.postInit");
+      super.postInit();
+    } else {
+      super.postInit();
+      writeln("B.postInit");
+    }
+  }
+}
+
+var b = new B(flag);
```

Comment:
I'd either add a .execopts that would test both values of this flag or, probably simpler, just create two objects of the type, one with B(true) one with B(false).